iron-swipeable-pages.html

```diff
@@ -253,6 +261,10 @@
 
         // prepare the page for animation and add it to the list of pages to be animated
         _initPage: function(page, transition, left) {
+          if (!page) {
+            return;
+          }
+
```

Comment:
this check is not necessary since _initPage should always be called with a proper argument